app/models/service_provider.rb

```diff
@@ -2,7 +2,7 @@
 require 'identity_validations'
 
 class ServiceProvider < ApplicationRecord
-  self.ignored_columns = %w[deal_id agency aal]
+  self.ignored_columns = %w[deal_id agency aal fingerprint]
```

Comment:
this was a very old column and adding it here makes sure we don't reference it at all

Reply:
> this was a very old column and adding it here makes sure we don't reference it at all

Appears we may still be referencing it during local development `make setup`:

https://github.com/18F/identity-idp/blob/5a0fb4c29ebb68a15b2e1df85e86075937ac1727/app/services/service_provider_seeder.rb#L20

https://github.com/18F/identity-idp/blob/5a0fb4c29ebb68a15b2e1df85e86075937ac1727/config/service_providers.localdev.yml#L320

This is preventing `make setup` from succeeding:

```
ActiveModel::UnknownAttributeError: unknown attribute 'fingerprint' for ServiceProvider.
/Users/Documents/Code/identity-idp/app/services/service_provider_seeder.rb:20:in `block in run'
/Users/Documents/Code/identity-idp/app/services/service_provider_seeder.rb:13:in `each'
/Users/Documents/Code/identity-idp/app/services/service_provider_seeder.rb:13:in `run'
/Users/Documents/Code/identity-idp/db/seeds.rb:2:in `<main>'
```

We do want to remove that configuration value from the second file?

Reply:
For posterity: #4877

Reply:
Also, should `fingerprint` be removed from `COLUMNS` in `NullServiceProvider`? Currently it behaves differently between `ServiceProvider` and `NullServiceProvider`:

```
[3] pry(main)> ServiceProvider.new.fingerprint
NoMethodError: undefined method `fingerprint' for #<ServiceProvider:0x00007fad4c15e810>
from /Users/andrewmduthie/.rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/activemodel-6.1.3.1/lib/active_model/attribute_methods.rb:469:in `method_missing'
[7] pry(main)> NullServiceProvider.new(issuer:nil).fingerprint
=> nil
```

https://github.com/18F/identity-idp/blob/e6fafb965a6414f5d1b0074d3c75daa5c06cab3e/app/models/null_service_provider.rb#L23

Reply:
thanks for these catches... will make a PR to remedy

Reply:
https://github.com/18F/identity-idp/pull/4884